testing-resources.md

```diff
@@ -0,0 +1,20 @@
+#Testing Resources and Tutorials
+
+Test driven development (TDD) means making tests that will ensure your programs do what you want, before you start to write any code.
+TDD helps us write stable code, and ensures that we can make changes without breaking our codes existing functionality. 
+It is also secretly what we are doing on Codewars!
+
+
+### Test Driven Development
+- [Agile Alliance Glossary](https://www.agilealliance.org/glossary/tdd/) Gives a more in depth explanation of Test Driven Development
+
+
+### Making Tests for Codewars
+- [Coding For Everyone - Write Your Own Kata](https://github.com/codingforeveryone/READMEs/blob/master/codewars/writing-your-own-kata.md) This includes a good guide to making tests for your Katas.
+- [Coding For Everyone - Random Tests](https://github.com/codingforeveryone/READMEs/blob/master/codewars/random-test-cases-for-complete-beginners.md) How to expand your testing suite with random tests.
+- [Conjured Codewars Codex - testing](https://bkaestner.github.io/codewars-rules/#tested-testing-theories) - What to do, and what not to do, when writing Codewars tests.
+- [Codewars Documentation - Javascript testing reference] https://www.codewars.com/docs/js-slash-coffeescript-test-reference - Codewars documentation. A good reference, but still a little confusing without a bit of background knowledge of testing frameworks.
```

Comment:
URL needs to be inside parentheses, and had 'Codewars Documentation' in the text twice.  Fixed below:

[Codewars Documentation - Javascript testing reference](https://www.codewars.com/docs/js-slash-coffeescript-test-reference) - A good reference, but still a little confusing without a bit of background knowledge of testing frameworks